build/makelib/golang.mk

```diff
@@ -48,7 +48,7 @@ GO_TEST_FLAGS ?=
 # ====================================================================================
 # Setup go environment
 
-GO_SUPPORTED_VERSIONS ?= 1.13|1.14
+GO_SUPPORTED_VERSIONS ?= 1.13|1.14|1.15
```

Comment:
Looks like you need to rebase on the latest. This commit shouldn't show up in your PR.

Reply:
Done.